macros/anaUltraScurve.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python
+#!/bin/env python2.7
 import os
```

Comment:
could I have the output of:

echo $PYTHONPATH

From one of the machines at p5 where this works with the default python (2.6.X?)